ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -130,17 +133,25 @@ private fun AppIcon(
     modifier: Modifier = Modifier,
 ) {
     val context = LocalContext.current
-    val appIconResId = remember {
-        val packageManager = context.packageManager
-        context.applicationInfo.loadIcon(packageManager)
-    }
 
-    Image(
-        bitmap = appIconResId.toBitmap(config = Bitmap.Config.ARGB_8888).asImageBitmap(),
-        contentDescription = null,
-        modifier = modifier,
-        contentScale = ContentScale.Crop,
-    )
+    if (LocalInspectionMode.current) {
```

Comment:
Should we reuse the `isInPreviewMode` composable? We might want to extract it to another file for that.